Python/tests/package_management_pypi_test.py

```diff
@@ -21,44 +24,55 @@ def _drop_all_ddl_packages(conn, scope):
                 except Exception as e:
                     pass
 
+# Get the list of packages installed before we do anything
+# 
 def _get_initial_list(conn, scope):
     pkgs = _get_sql_package_table(conn)
     return pkgs['name']
 
-pyexecutor = SQLPythonExecutor(connection)
-pkgmanager = SQLPackageManager(connection)
-initial_list = _get_sql_package_table(connection)['name']
-
+# Check if a package exists
+#
 def _package_exists(module_name: str):
     mod = __import__(module_name)
     return mod is not None
 
-
+# Check that a package does NOT exist
+# 
 def _package_no_exist(module_name: str):
     import pytest
     with pytest.raises(Exception):
         __import__(module_name)
     return True
 
-@pytest.mark.skip(reason="No version of tensorflow works with currently installed numpy (1.15.4)")
-def test_install_tensorflow():
-    def use_tensorflow():
-        import tensorflow as tf
-        node1 = tf.constant(3.0, tf.float32)
-        return str(node1.dtype)
-    
+
+pyexecutor = SQLPythonExecutor(connection)
+pkgmanager = SQLPackageManager(connection)
+initial_list = _get_sql_package_table(connection)['name']

```

Comment:
Should probably move these 3 lines up to the beginning of the file.  initial_list is used above.